packages/wonder-blocks-modal/docs.md

```diff
@@ -1,4 +1,209 @@
-Looking for docs for StandardModal, OneColumnModal, or TwoColumnModal click [here](https://deploy-preview-389--wonder-blocks.netlify.com/#modal)
+Looking for docs for StandardModal, OneColumnModal, or TwoColumnModal click
+[here](https://deploy-preview-389--wonder-blocks.netlify.com/#modal)
+
+## Examples
+
+### Example: Default modal
+
+Once the modal is launched, tab focus wraps inside the modal content. Pressing Tab at the end of the modal will focus the modal's first element, and pressing Shift-Tab at the start of the modal will focus the modal's last element.
+
+```js
+const {StyleSheet} = require("aphrodite");
```

Comment:
I assume there will be some merge conflicts with your import diff here.

Reply:
I created the import diff based on this PR ....and I forgot to mention that in the other PR :|